src/drivers/dw/dma.c

```diff
@@ -1322,6 +1322,22 @@ static int dw_dma_get_data_size(struct dma *dma, int channel, uint32_t *avail,
 	return 0;
 }
 
+static int dw_get_attribute(struct dma *dma, uint32_t type,
+			    uint32_t *value)
+{
+	int ret = 0;
+
+	switch (type) {
+	case DMA_ATTR_ALIGNMENT:
+		*value = DW_DMA_BUFFER_ALIGNMENT;
+		break;
+	default:
+		ret = -EINVAL;
```

Comment:
It's good practice to add break also after default label.

Reply:
ok